lib/plugins/inspec-habitat/templates/habitat/hooks/run.erb

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+exec 2>&1
+
+CONFIG="{{pkg.svc_config_path}}/inspec_exec_config.json"
```

Comment:
Perhaps I am dense, but is `{{ ... }}` ERB syntax?

Reply:
Nope, in this case it is Handlebar and is rendered in Habitat during build.